src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition2Test.java

```diff
@@ -224,4 +224,11 @@ public void traitsSandbox() throws Exception {
         jenkins.assertLogContains("org.jenkinsci.plugins.scriptsecurity.sandbox.RejectedAccessException: Scripts not permitted to use staticMethod jenkins.model.Jenkins getInstance", b);
     }
 
+    @Issue("JENKINS-46391")
+    @Test
+    public void tildePattern() throws Exception {
```

Comment:
Not a task for this PR, but sometime this test suite needs to be split up and rearranged more logically, since of course it is not testing `CpsFlowDefinition` per se.